src/app/public/src/modules/page-anchor/page-anchor.component.scss

```diff
@@ -26,3 +26,8 @@
   height: 0.65em;
   transition: opacity 300ms;
 }
+
+.anchor-with-omnibar {
+  padding-top: 5em;
+  margin-top: -3em;
```

Comment:
This is brilliant!

I'm not sure you need the `.anchor-with-omnibar` class, though. You can just put this directly on the `.stache-page-anchor` class and save yourself the component conditional.

Reply:
Also, I'd double-check that this behavior works the same way on Windows browsers :)

Reply:
I added the class because without the omnibar it scrolled to the anchor, but a bit high, around 45px higher than seemed normal.

Reply:
@Blackbaud-SteveBrush confirmed, it does work in IE

Reply:
Another idea (and this will also allow you to remove the omnibar logic in this component), you could use the `body` class in the stylesheet:

```
.stache-page-anchor {
  padding-top: 2em;
  margin: 0 0 1em;
  padding-bottom: 0.6em;
  border-bottom: 1px solid $stache-border-color-neutral-light;

  &:hover {
    .stache-page-anchor-icon {
      opacity: 1;
    }
  }

  /deep/
  .stache-omnibar-enabled & {
    padding-top: 5em !important;
  }
}
```

The above will at least get you to feature-parity, without the extra logic in the component. However, I'm curious about something else: It would be ideal if the top padding/margin of the Page Anchors was consistent whether or not the Omnibar was present. I know you mentioned the smooth-scroll method earlier. Did you discover a potential solution with that?

Reply:
The solution with the smooth-scroll stuff was to adjust the padding and go negative margin.  The scroll goes to the top of the element (padding, not margin).  There is no way to set an offset of pixels or the like.  If we kept the padding/margin the same with or without omnibar, the element specified would just show up farther down the page than it would appear with the omni bar enabled.   If that's acceptable we can just always do the positive padding negative margin.

Reply:
Thanks for the explanation. I think if you can use the `body` class, along with the positive/negative padding/margin, this would work great! (Updated example above.)